<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorDialog</class>
 <widget class="QDialog" name="SensorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SensorDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLyse">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Lyse</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbWaste">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Waste</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDiluent">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Diluent</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLyse">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>HAS LYSE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leWaste">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>NOT FULL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDiluent">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>HAS DILUENT</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
